### PR TITLE
fixing some versioning issues

### PR DESCRIPTION
**Pull request by alanvoss**
[Card](https://rentpath.leankit.com/card/607001126)

### CODE REVIEW

File: rpenv.spec [hunk 1]
```diff
@@ -39,6 +39,13 @@ rm -rf %{buildroot}
 
 
 %changelog
+* Wed Aug 03 2016 Brad Anderson <avoss@rentpath.com> - 3.0.1
```
Comment (rkledesma) [inline rpenv.spec]: Why are the timestamps dated in 2016?

Comment (pasha) [reply rpenv.spec]: Also name and emails do not match, line 45 too.

Comment (dgmcguire) [reply rpenv.spec]: Should we be keeping a changelog in the file?  Maybe something else depends on it, but it seems like git is much better suited for keeping up with commit history

Comment (pasha) [reply rpenv.spec]: 👍 `git is much better suited for keeping up with commit history`

Comment (dgmcguire) [reply rpenv.spec]: the spec file is for RPM - but if this repo is public the RPM users will still be able to find the github repo

**Comment by Christopher-Bui** [comment]
The spec file is for the RPM build infra does.

**Comment by pasha** [comment]
@Christopher-Bui do you know if `changelog` needs to be a part of it?

**Comment by alanvoss** [comment]
I put it back, @pasha

**Pull request merged**